events/urls.py

```diff
@@ -1,3 +1,8 @@
 from django.urls import path
 
-urlpatterns = []
+from .views import create_event, event_list
+
+urlpatterns = [
+    path('create/', create_event, name='create_event'),
+    path('listing/', event_list, name='events_list'),
```

Comment:
it's crucial to be consistent
for example either it should be create_event, list_events or event_create, events_list,
first pair is the right one BTW